canonical-web-and-design/ubuntu.com
# Observability build

## Done

- [x] Built Oberservability page

- [x] Built Observability form

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/observability or in the demo link: https://ubuntu-com-10105.demos.haus/observability
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare page against [copy-doc](https://docs.google.com/document/d/1NxJdT3BD6EWnpsu-eathNyranQVImnDGhuCjzsIWh4s/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4242
